requirements.txt

```diff
@@ -20,8 +20,6 @@ Pillow==9.2.0
 protobuf==3.20.1
 pyparsing==3.0.9
 python-dateutil==2.8.2
-psycopg2==2.9.3
```

Comment:
`psycopg2` is needed, test your code before commiting or I will add a github action on this repo.
The unneeded is socketio related packages.